### ISSUE TITLE
How to load cues from a Plugin Content folder path?

### ISSUE THREAD

**Issue by EnricoPietrocola** [tranek/GASDocumentation#120]
I have read from the Cue Manager section in the documentation that a custom path for loading GA cues can be added to DefaultGame.ini, but it doesn't seem to work with plugins. I get this error when setting any path originating from /Plugins/ 
`" ScanPathsSynchronous: /Plugins/<path> is not in a mounted path, will not scan"`

I'd like to have my cues in my plugin content folder with all the rest because my core gameplay features are in a plugin I use to quickly port my characters/gameplay features to projects

**Comment by EnricoPietrocola** [comment]
P.s. Using a path such as `"/EonFramework/Content/Eon/Core/Abilities/"` , without the `Plugins` before doesn't seem to trigger the warning mentioned, cues are still not being launched though

**Comment by EnricoPietrocola** [comment]
Turns out the correct path was `/EonFramework/Eon/Core/Abilities/`, which is something I can't understand of where it's coming from, since both project folder structure and the UE path don't seem to compare. but I can now see my cues again while they're in the plugin subfolder I wanted.

For the other who might encounter the issue, in order to find the path I copied a Cue blueprint, I have pasted in the folder I wanted it to be, an error prompts telling you that cues only work if set in the path specified. I then edited my `+GameplayCueNotifyPaths=` key to that path and it started working (after a restart of the editor clearly).
